package.json

```diff
@@ -15,6 +15,7 @@
     "lint:es": "eslint . --report-unused-disable-directives",
     "lint:js": "npm run lint:es",
     "lint:html": "npm run htmlhint && npm run vnu-jar",
+    "lint:links": "linkinator aria-practices.html && linkinator examples/ --recurse",
```

Comment:
Doesn't look like combining the globs works right. It didn't find the same number of links and had some random failures for `.\` links that don't exist.
I can see that shortcurcuiting in the current one could cause the need to push twice to catch all the issues, but I'm not sure that's a big issue.
Could also add a lint-staged target though if you think it would help

Reply:
> Doesn't look like combining the globs works right. It didn't find the same number of links and had some random failures for `.\` links that don't exist.

Okay, that's unfortunate. Odd behavior.

> I can see that shortcurcuiting in the current one could cause the need to push twice to catch all the issues, but I'm not sure that's a big issue.

Fair enough, and it may not happen often enough to be even be an issue.

> Could also add a lint-staged target though if you think it would help

No need. Looks good to me!